### DISCUSSION TITLE
Using van in my upcoming js13k challenge

### DISCUSSION THREAD

**Discussion by benallfree** [vanjs-org/van#69] [Show and tell]
Just thought I'd drop a note to the community that van is being used in https://github.com/benallfree/worldforge, my collaborative open game engine I'm developing for js13k. I first developed it in Svelte, but that became too cumbersome. Van adds very little overhead and gives me reactive programming and state management. It's perfect.

**Comment by Tao-VanJS** [comment]
Glad to hear that VanJS can be helpful here! 😃 You can keep us posted about your progress and achievement! 🚀🚀🚀

**Comment by artydev** [comment]
Great :-)
